src/Instagram.php

```diff
@@ -525,16 +548,14 @@ public function pagination($obj, $limit = 0)
             }
 
             $function = str_replace(self::API_URL, '', $apiCall[0]);
+            $count = ($limit) ? $limit : count($obj->data);
 
-            $auth = (strpos($apiCall[1], 'access_token') !== false);
-
-            if (isset($obj->pagination->next_max_id)) {
-                return $this->_makeCall($function, $auth, array('max_id' => $obj->pagination->next_max_id, 'count' => $limit));
+            if (isset($obj->pagination->next_max_tag_id)) {
+                return $this->_makeCall($function, array('max_tag_id' => $obj->pagination->next_max_tag_id, 'count' => $count));
             }
 
-            return $this->_makeCall($function, $auth, array('cursor' => $obj->pagination->next_cursor, 'count' => $limit));
+            return $this->_makeCall($function, array('next_max_id' => $obj->pagination->next_max_id, 'count' => $count));
```

Comment:
The pagination call is made with a parameter named "next_max_id". It seems that it should be named "max_id". For example, see the parameters for the ["/users/self/media/recent/" endpoint](https://www.instagram.com/developer/endpoints/users/#get_users_media_recent_self). Am I missing something?